File: java/suyash sir/lastdigit.java
import java.util.Scanner;

public class lastdigit {
    public static void main(String[] args) {
        System.out.println("Enter digits");
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();

        if (a % 10 == 4) {
            System.out.println("Yes last digit is 4");
        } else {
            System.out.println("Not");
        }
    }
}

File: c language/suyash sir/lastdigit.c
#include <stdio.h>
int main(){
int a,b;
printf("Enter number: ");
scanf("%d",&a);
printf("Enter check number: ");
scanf("%d",&b);
if (a%10==b)
{
   printf("Last digit of the number is %d",b);
}
else
{
    printf("NOT");
}


    return 0;
}